sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_differentiate_output_models_trained_with_and_without_labels_async.py

```diff
@@ -74,6 +75,9 @@ async def recognize_custom_forms(self):
                         format_bounding_box(field.value_data.bounding_box),
                         field.confidence
                     ))
+            print("\nFind the value for a specific labeled field using the training-time label:")
+            for labeled_form in forms_with_labeled_model:
+                print("The Merchant is {}\n".format(labeled_form.fields["Merchant"].value))
```

Comment:
Are these keys ("Merchant") specific to the form that is used in this particular sample?

Reply:
Yes, just fields I picked at random. But you have a point - for the labeled one i'm just guessing that a training-time label "Merchant" exists. I'll update with comments to be a bit more clear.